pacemaker.spec

```diff
@@ -143,7 +143,7 @@
 Name:          pacemaker
 Summary:       Scalable High-Availability cluster resource manager
 Version:       %{pcmkversion}
-Release:       %{pcmk_release}%{?dist}
+Release:       %{pcmk_release}%{?dist}.0.0.rdo1
```

Comment:
There is already pacemaker 1.1.16-12.el7_4.2  in 7.4 ZStream, does this one include all the required fixes?

Reply:
Meh I did not notice that older zStream came out already. It has some completely unrelated fixes. I will just push a change for the 'Release:' pointing to %{pcmk_release}%{?dist}.2.0.0.rdo1. That means that in the RDO package we will miss the few patches that came in with the 12.2 zstream but that is totally okay (also discussed this with beekhof and he's fine with this plan). Reason for it is that the patches we have here will need a bit of a rebase if we go for the 12.2 zstream + our patches. Ken will get to it for the upcoming next zStream (Mid-Oct), but the timing is just not right for this temporary work.

Reply:
I have pushed the change. But I think it is still a bit of an issue:
$ rpmdev-vercmp 1.1.16-12.el7_4.2  1.1.16-12.el7.2.0.0.rdo1 
1.1.16-12.el7_4.2 > 1.1.16-12.el7.2.0.0.rdo1

Even if I set it to .3 we're still screwed:
$ rpmdev-vercmp 1.1.16-12.el7_4.2  1.1.16-12.el7.3.0.0.rdo1                                                                                                                            1.1.16-12.el7_4.2 > 1.1.16-12.el7.3.0.0.rdo1

Is there a way out from this?

Reply:
The only way out I can think of, is that for this specific build we somehow force %{dist} to be .el7_4  instead of .el7. I guess if it is not overridable in CBS we could hardcode it here for this temporary solution?

Reply:
That is what I did in the last commit. Let me know what you think :)